### PR TITLE
[CBI-1831] Enable Cere -> Ethereum flow

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
- Updated config
- Updated chain config
- Removed destinationchain config parameter from chainbridge config
- Updated polkadot/api version
- Updated UI for choosing the wallet type.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## vNext
 
-- ...
+- Update config, chain-spec file to enable Cere native to Cere ERC20 transfer
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md]: Just use `Enable CERE native to CERE ERC20 transfers`. It is clear from the diff what has been changed.

File: config/chainbridge-runtime-config.cere.dev.js [hunk 2]
```diff
@@ -38,8 +38,8 @@ window.__RUNTIME_CONFIG__ = {
         type: "Substrate",
         nativeTokenSymbol: "CERE",
         chainbridgePalletName: "chainBridge",
-        bridgeFeeFunctionName: "tokenTransferFee",
-        transferPalletName: "palletBridge",
+        bridgeFeeValue: 0,
```
Comment (AndreiNavoichyk) [inline config/chainbridge-runtime-config.cere.dev.js:41]: What is the function `tokenTransferFee`? Why did you remove it?

File: config/chainbridge-runtime-config.cere.dev.js [hunk 3]
```diff
@@ -52,16 +52,15 @@ window.__RUNTIME_CONFIG__ = {
...
         ],
-        destinationChain: true,
       },
       {
-        chainId: 5,
+        chainId: 0,
         networkId: 5,
         name: "Ethereum Goerli",
         decimals: cereTokenDecimals,
+        rpcUrl: "https://goerli.infura.io/v3/2532f2073c2d4a34b5b9544c76a3de8b ",
```
Comment (AndreiNavoichyk) [inline config/chainbridge-runtime-config.cere.dev.js]: Which Infura project does it belong to?

File: config/chainbridge-runtime-config.cere.dev.js [hunk 4]
```diff
@@ -52,16 +52,15 @@ window.__RUNTIME_CONFIG__ = {
             decimals: cereTokenDecimals,
           },
         ],
-        destinationChain: true,
       },
       {
-        chainId: 5,
+        chainId: 0,
```
Comment (AndreiNavoichyk) [inline config/chainbridge-runtime-config.cere.dev.js:57]: Why did you change?

File: src/Components/Pages/TransferPage.tsx [hunk 5]
```diff
@@ -360,7 +360,7 @@ const TransferPage = () => {
             return parseFloat(value) <= tokens[preflightDetails.token].balance;
           } else {
             return (
-              parseFloat(value + (bridgeFee || 0)) <=
+              parseFloat(value) + (bridgeFee || 0) <=
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx:363]: Why?

File: src/Components/Pages/TransferPage.tsx [hunk 6]
```diff
@@ -377,7 +377,8 @@ const TransferPage = () => {
               preflightDetails.token,
               destinationChainConfig.chainId
             );
-            return Boolean(supplies);
+            // return Boolean(supplies);
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: You can not hardcode the result here.

File: src/Contexts/NetworkManagerContext.tsx [hunk 7]
```diff
@@ -136,14 +136,14 @@ const NetworkManagerProvider = ({ children }: INetworkManagerProviderProps) => {
         setDestinationChains(
           chainbridgeConfig.chains.filter(
             (bridgeConfig: BridgeConfig) =>
-              bridgeConfig.destinationChain === true
+              bridgeConfig.chainId !== chain.chainId
```
Comment (AndreiNavoichyk) [inline src/Contexts/NetworkManagerContext.tsx:139]: How will it work for user now?

File: src/Modules/NetworkSelectModal.tsx [hunk 8]
```diff
@@ -8,10 +8,22 @@ import {
   ProgressBar,
   Typography,
 } from "@chainsafe/common-components";
+import styles from "../Constants/constants";
```
Comment (AndreiNavoichyk) [inline src/Modules/NetworkSelectModal.tsx]: Let's exclude redesign from this PR. We need to set up the current UI to support Cere -> Ethereum. Only minimal changes.

File: config/chainbridge-runtime-config.cere.dev.js (hunk 2 above)
Comment (karthikeyan-cere) [reply config/chainbridge-runtime-config.cere.dev.js:41]: We don't have bridge fee implementation in our pallet. So it gives us the error when it tries to fetch the bridge fee via the pallet, to override this issue I have removed the `bridgeFeeFunctionName` and introduced the `bridgeFeeValue` variable, So now we have defined the bridge fee value in config it will skip fetching from pallet.

File: config/chainbridge-runtime-config.cere.dev.js (hunk 4 above)
Comment (karthikeyan-cere) [reply config/chainbridge-runtime-config.cere.dev.js:57]: In our project, we have configured the 0 as the id for Ethereum Blockchain.

File: src/Components/Pages/TransferPage.tsx (hunk 5 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx:363]: The value is returned as a string, so it concatenates with the bridge value. To overcome this issue, I have converted the type of value and then added it to the bridge fee.

File: src/Contexts/NetworkManagerContext.tsx (hunk 7 above)
Comment (karthikeyan-cere) [reply src/Contexts/NetworkManagerContext.tsx:139]: As we don't have the same chain Id for the two blockchains, it will work as expected. The blockchain which has been set as home chain wont be shown under the destination chain.

File: src/Modules/NetworkSelectModal.tsx (hunk 8 above)
Comment (karthikeyan-cere) [reply src/Modules/NetworkSelectModal.tsx]: I have introduced this small change in this PR because the current button hides while hovering the mouse over it.

File: config/chainbridge-runtime-config.cere.dev.js (hunk 3 above)
Comment (karthikeyan-cere) [reply config/chainbridge-runtime-config.cere.dev.js]: Network-relayer dev.

File: src/Components/Pages/TransferPage.tsx (hunk 6 above)
Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: updated the logic

File: CHANGELOG.md (hunk 1 above)
Comment (karthikeyan-cere) [reply CHANGELOG.md]: Updated the changelog

**Review by AndreiNavoichyk — APPROVED** [review]
